server/node_test.go

```diff
@@ -102,14 +102,16 @@ func TestBootstrapCluster(t *testing.T) {
 
 	// Scan the complete contents of the local database.
 	sr := &proto.ScanResponse{}
-	if err := localDB.Call(proto.Scan, &proto.ScanRequest{
-		RequestHeader: proto.RequestHeader{
-			Key:    engine.KeyLocalPrefix.PrefixEnd(), // skip local keys
-			EndKey: engine.KeyMax,
-			User:   storage.UserRoot,
+	if err := localDB.Run(&client.Call{
```

Comment:
See my other comment. I need to do another pass to normalize all the client calls. I'd really prefer to do this in a separate change. This one is already unwieldy enough.